src/main/java/teammates/storage/entity/Comment.java

```diff
@@ -98,6 +106,8 @@ public Comment(String courseId, String giverEmail, CommentRecipientType recipien
         this.showRecipientNameTo = showRecipientNameTo;
         this.createdAt = date;
         this.commentText = comment;
+        this.lastEditorEmail = (lastEditorEmail == null) ? giverEmail : lastEditorEmail;
+        this.lastEditedAt = (lastEditedAt == null) ? date : lastEditedAt;
```

Comment:
ref: https://github.com/TEAMMATES/repo/blob/master/devdocs/design.md#storage
you can get the ideas about entity and DTO